echopype/convert/parse_azfp.py

```diff
@@ -197,6 +197,25 @@ def _compute_battery(self, ping_num, battery_type):
 
         return N * USL5_BAT_CONSTANT
 
+    def _computer_pressure(self, ping_num, is_valid):
```

Comment:
Rename the function:
```suggestion
    def _compute_pressure(self, ping_num, is_valid):
```

Reply:
Resolved.